### PR TITLE
Add Goerli 1.9.5

### PR DESCRIPTION
**Pull request by gbalabasquer**

### CODE REVIEW

**Review by godsflaw — DISMISSED** [review]
LGTM, but the link to staging at the top should be updated when we get staging portals that point to goerli.  For now, we have to start somewhere.

**Comment by gbalabasquer** [comment]
Link to kovan UIs should be changed to goerli ones as soon as they are implemented.

File: releases/goerli/1.9.5/index.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+	</head>
+	<body>
+		<div>
+			<h2>
+				<strong>Goerli Release 1.9.5</strong>
+			</h2>
+
+			<ul>
+				<li><a href="/releases/goerli/1.9.5/contracts.json">Contract addresses</a></li>
+				<li><a href="https://ipfs.io/ipfs/QmcHJSEKK5jEoVMNFqfujht33ZL75r5bEbpsp6URBhmRLT">Contract addresses (IPFS hosted)</a></li>
```
Comment (zdumitru) [inline releases/goerli/1.9.5/index.html:13]: this doesn't match the contracts in the PR.

**Review by zdumitru — CHANGES_REQUESTED** [review]

File: releases/goerli/1.9.5/index.html (hunk 1 above)
Comment (gbalabasquer) [reply releases/goerli/1.9.5/index.html]: Good catch I had forgotten to update the RWAs PIPs. It's replaced now.

**Review by zdumitru — APPROVED** [review]

**Comment by brianmcmichael** [comment]
Verified on-chain changelog has been updated with the new addresses since we're re-purposing that from an earlier deployment. :+1:

**Review by brianmcmichael — APPROVED** [review]

**Pull request merged**